templates/vm_mount@.service.j2

```diff
@@ -0,0 +1,19 @@
+# {{ ansible_managed }}
+[Unit]
+Description             = Mounts rootfs for vm %i
+Documentation           = https://github.com/stuvusIT/xen_vman
+BindsTo                 = vm_iscsi@%i.service
+After                   = vm_iscsi@%i.service
+Conflicts               = vm@%i.service
+ConditionPathExists     = {{ xen_vman_vm_config_path }}/%i.cfg
+
+[Service]
+Type                    = oneshot
+RemainAfterExit         = yes
+ExecStart               = /etc/xen/scripts/mount_vm.sh -s %i -m
+ExecStop                = /etc/xen/scripts/mount_vm.sh -s %i -u
+TimeoutStartSec         = 20
```

Comment:
There is a timeout in `mount_vm`. If it is larger than this value, the script will be killed too early.